test/SDK/SmartDetectorsSDKTests/ResourceIdentifierTests.cs

```diff
@@ -165,6 +196,14 @@ public void WhenConvertingVmResourceTheConversionIsSuccessful()
             VerifyConversion(testResourceId, testResourceIdentifier);
         }
 
+        [TestMethod]
+        public void WhenConvertingAKSResourceTheConversionIsSuccessful()
```

Comment:
>AKS [](start = 34, length = 3)

Only two letter acronyms should be all upper case.
So either Aks or KubernetesService (better)